doc/options.rst

```diff
@@ -0,0 +1,115 @@
+Options and file formats
+------------------------
+In the optimization-process of *SMAC*, there are several ways to configure the
+options:
+
+Mandatory:
+    * `Commandline <basic_usage.html#commandline>`_-options, with which *SMAC* is called directly (not needed if
+      *SMAC* is used within Python).
+    * Scenario-options, that are specified via a Scenario-object. Either directly
+      in the Python-code or by using a scenario_-file.
+    * A Parameter Configuration Space (`PCS <options.html#paramcs>`_), that provides the legal ranges of
+      parameters to optimize, their types (e.g. int or float) and their default
+      values.
+
+Optional:
+    * Instance_- and feature_-files, that list the instances and features to
+      optimize upon.
+
+.. _scenario:
+
+Scenario
+~~~~~~~~
+The scenario-object is used to configure *SMAC* and can be constructed either by providing an actual
+scenario-object (see `SVM-example <quickstart.html#using-smac-in-python-svm>`_), or by specifing the options in a
+scenario file (see `SPEAR example <quickstart.html#spear-example>`_).
+The format of the scenario file is one option per line:
+
+.. code-block:: bash
+
+        OPTION1 = VALUE1
+        OPTION2 = VALUE2
+        ...
+
+For boolean options "1" or "true" both evaluate to True.
+The following assumes that the scenario is created via a scenario-file. If it is
+generated within custom code, you might not need *algo* or *paramfile*.
+
+Required:
```

Comment:
Would it be possible to dynamically create this list of arguments from the scenario file?